prelexer.cpp

```diff
@@ -423,6 +423,9 @@ namespace Sass {
     const char* number(const char* src) {
       return sequence< optional<sign>, unsigned_number>(src);
     }
+    const char* sass_number(const char* src) {
+      return sequence< optional<sign>, unsigned_number, optional < identifier > >(src);
```

Comment:
Yes, that would be great. Let me know if you find something usefull in their source!